anecdote/assets/logs/anecdote.yaml

```diff
@@ -0,0 +1,50 @@
+id: anecdote
+metric_id: anecdote
+backend_only: true
+pipeline:
+  type: pipeline
+  name: Anecdote Log Pipeline
+  enabled: true
+  filter:
+    query: "source:anecdote"
+  processors:
+    - type: service-remapper
+      name: Define `service` as the official service of the log
+      enabled: true
+      sources:
+        - service
```

Comment:
I did some test in staging, and I don't have the box for the `service-remapper`. The source from the samples you provided seems to indicate it should be: `content.service` the source and not `service`.

From what I can see, the issue if one the two:
- either you are providing as well the service in your app, in that case you can do this as well in the sample doing [like here](https://github.com/DataDog/integrations-core/blob/70f798dc68201e36078fa5b57c29308436c339a0/cisco_duo/assets/logs/cisco-duo_tests.yaml#L47). (Please note the `service: <service-name>` is outside of the sample payload)
- you can edit the source here to use `content.service` instead.

Once fix, I am good to approve this PR!

Sample example I submitted for this integration:
<img width="959" alt="image" src="https://github.com/user-attachments/assets/28eb17da-3f10-4d6c-8c0e-a885fdb478c2">

Reply:
What is should like with the service-remapper properly set:
<img width="695" alt="image" src="https://github.com/user-attachments/assets/c2b5fc97-fe6f-459b-80c1-5f6b79254054">

Reply:
Hey @thibaultkrebs, 
Thank you for your review, I updated log pipeline as you suggested, could you check it, please?

Reply:
@atyulyutaev https://github.com/DataDog/integrations-extras/pull/2347/files#diff-c18616386657bda46623cfbb38372a680afe186c3d6762fb1d0c4b8a6f011027

You just need to fix the message and the test sample with the new mapping and we are good to go!

Reply:
Hey,
I fixed the samples, could you check it please?

Reply:
All good for me now, thank you! (approved)